src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_LENGTH = 6;
+
+    public static void main(String[] args) {
+        Figure[] figuresArray = new Figure[ARRAY_LENGTH];
+        FigureSupplier supplier = new FigureSupplier();
+
+        for (int i = 0; i < figuresArray.length; i++) {
+            if (i <= figuresArray.length / 2) {
+                figuresArray[i] = supplier.getRandomFigure();
+            }
+            figuresArray[i] = supplier.getDefaultFigure();

```

Comment:
```suggestion
            if (i <= figuresArray.length / 2) {
                figuresArray[i] = supplier.getRandomFigure();
            } else {
                figuresArray[i] = supplier.getDefaultFigure();
            }

```

Otherwise, the figure will be overwritten each time, since it will reach here anyway. And we need either or